scripts/test

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+[ -e venv ] && source venv/bin/activate
+
+export PYTHONPATH=.
+
+# The unittest discover feature somehow doesn't pick up all tests
+# python -m unittest discover -s tests
+find tests -name 'test_*.py' -exec python3 -m unittest -v {} \;
```

Comment:
We were actually using `nosetests` to run the tests, it does all the recursive file detection and path setting. Probably would make sense to add it to the readme somewhere :)